client/rpc-api/src/system/error.rs

```diff
@@ -45,17 +51,16 @@ const NOT_HEALTHY_ERROR: i32 = BASE_ERROR + 1;
 // Peer argument is malformatted.
 const MALFORMATTED_PEER_ARG_ERROR: i32 = BASE_ERROR + 2;
 
-impl From<Error> for JsonRpseeError {
-	fn from(e: Error) -> Self {
+impl From<Error> for ErrorObjectOwned {
+	fn from(e: Error) -> ErrorObjectOwned {
 		match e {
 			Error::NotHealthy(ref h) =>
-				CallError::Custom(ErrorObject::owned(NOT_HEALTHY_ERROR, e.to_string(), Some(h))),
-			Error::MalformattedPeerArg(e) => CallError::Custom(ErrorObject::owned(
-				MALFORMATTED_PEER_ARG_ERROR + 2,
-				e,
-				None::<()>,
-			)),
+				ErrorObject::owned(NOT_HEALTHY_ERROR, e.to_string(), Some(h)),
+			Error::MalformattedPeerArg(e) =>
+				ErrorObject::owned(MALFORMATTED_PEER_ARG_ERROR + 2, e, None::<()>),
```

Comment:
`MALFORMATTED_PEER_ARG_ERROR` is already `BASE_ERROR + 2;` on ln 52.

Just checking it is intentional to bump it another `2` here.

Reply:
good catch thanks,